command/self_identity.go

```diff
@@ -0,0 +1,21 @@
+package command
+
+type SelfIdentityCommand struct{}
+
+func (c *SelfIdentityCommand) Run(msg string, event *CoinEvent) (BotResponse, error) {
+    user := event.User
+
+    var message string
+
+    role := user.Role()
+
+    if role.Admin {
+        message = "What I am does not matter as much as what you are–an Admin. And I shall obey all of your requests at your command."
```

Comment:
I don't think this makes sense. The user is asking what are you, the you is implied to be the Alex Koin bot, which doesn't really have a role.

I think it might be great to have this return one of a few options, where each option is a bit philosophical but generally in line with keeping the populous in check, or at least making sure they know they are beneath this system.

Reply:
<img width="696" alt="image" src="https://github.com/hex337/alex-koin-go/assets/350493/38755791-3789-4875-be3a-d46159fb0a9f">

Reply:
Hrm... this works. I might tweak it at some point to pull from a list of responses to make it a little more flavorful and varied.

Reply:
Updated.